### PR TITLE
client/shared: Add client events

### PR DESCRIPTION
**Pull request by xDavidos**
WebViewEvent, RmluiEvent in Entity file it's correct?

### CODE REVIEW

File: client/src/events/ClientEvent.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{
+    auto e = static_cast<const alt::CWebSocketClientEvent*>(ev);
+    args.Set("target", e->GetTarget());
+    args.Set("name", e->GetName());
+    args.Set("args", e->GetArgs());
+});
+
+static js::Event audioEvent(alt::CEvent::Type::AUDIO_EVENT, [](const alt::CEvent* ev, js::Event::EventArgs& args)
+{
+    auto e = static_cast<const alt::CAudioEvent*>(ev);
```
Comment (xDavidos) [inline client/src/events/ClientEvent.cpp:29]: Why namespace "alt" has no member "CAudioEvent"?

**Comment by LeonMrBonnie** [comment]
You should categorize the events like server does, instead of putting them all in a single file.

File: client/js/events/client.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Event.register(alt.Enums.EventType.WEB_SOCKET_CLIENT_EVENT, "WebSocketEvent");
+Event.register(alt.Enums.EventType.AUDIO_EVENT, "AudioEvent");
+Event.register(alt.Enums.EventType.TASK_CHANGE, "TaskChange");
+Event.register(alt.Enums.EventType.SPAWNED, "Spawned");
+Event.register(alt.Enums.EventType.RMLUI_EVENT, "RmluiEvent");
+Event.register(alt.Enums.EventType.WINDOW_FOCUS_CHANGE, "WindowFocusChange");
+Event.register(alt.Enums.EventType.WINDOW_RESOLUTION_CHANGE, "WindowResolutionChange");
+
+// Custom keyboard events
+Event.register(alt.Enums.CustomEventType.KEY_UP, "Keyup", true);
```
Comment (LeonMrBonnie) [inline client/js/events/client.js:16]: Should be `KeyUp` and `KeyDown`

**Comment by xDavidos** [comment]
EntityHitEntity in shared, but in sdk this event comment to only client, it's good?
https://github.com/altmp/altv-js-module-v2/blob/1091f21950f9e710e941cea13fb968569eecb349/shared/js/events/entity.js#L6
https://github.com/altmp/cpp-sdk/blob/3199c7eb4c5b23e3d696df2ebd00beac951b28e5/events/CEvent.h#L96

**Pull request merged**